<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>758</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FaceSwap</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="1">
     <widget class="QSlider" name="textureAlphaSlider">
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="value">
       <number>128</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QSlider" name="smoothTransitionSlider">
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="value">
       <number>255</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QSlider" name="shapeAlphaSlider">
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="value">
       <number>255</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Smooth transition</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="AnnotatorWidget" name="graphicsView_1"/>
       <widget class="AnnotatorWidget" name="graphicsView_2"/>
      </widget>
      <widget class="ImageWidget" name="morphedImage"/>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Texture</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Shape</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QCheckBox" name="colorMatchCheckBox">
      <property name="text">
       <string>Color match</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>758</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFIle">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoad_project"/>
    <addaction name="actionSave_project"/>
   </widget>
   <widget class="QMenu" name="menuImage">
    <property name="title">
     <string>Image</string>
    </property>
    <addaction name="actionLoadImage_1"/>
    <addaction name="actionLoadImage_2"/>
   </widget>
   <addaction name="menuFIle"/>
   <addaction name="menuImage"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionLoad_project">
   <property name="text">
    <string>Load project</string>
   </property>
  </action>
  <action name="actionSave_project">
   <property name="text">
    <string>Save project</string>
   </property>
  </action>
  <action name="actionLoadImage_1">
   <property name="text">
    <string>LoadImage 1</string>
   </property>
  </action>
  <action name="actionLoadImage_2">
   <property name="text">
    <string>LoadImage 2</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AnnotatorWidget</class>
   <extends>QGraphicsView</extends>
   <header>AnnotatorWidget.h</header>
  </customwidget>
  <customwidget>
   <class>ImageWidget</class>
   <extends>QGraphicsView</extends>
   <header>ImageWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
